class-vip-support-user.php

```diff
@@ -93,6 +93,17 @@ class User {
 	 */
 	const VIP_SUPPORT_USER_META_KEY = '_vip_support_user';
 
+	/**
+	 * Base email address for VIP Support user aliases.
+	 */
+	const VIP_SUPPORT_EMAIL_ADDRESS = 'vip-support@automattic.com';
+
+	/**
+	 * Regex Pattern for `VIP_SUPPORT_EMAIL_ADDRESS` to match aliases like:
+	 * `vip-support+<username>@automattic.com`
+	 */
+	const VIP_SUPPORT_EMAIL_ADDRESS_PATTERN = '/vip-support\+(.*)@automattic\.com/i';
```

Comment:
Minor: we can make the regex a bit more specific and avoid possible complications that can come with `.*`:

```suggestion
	const VIP_SUPPORT_EMAIL_ADDRESS_PATTERN = '/vip-support\+[^@]+@automattic\.com/i';
```

Reply:
Thanks! Updated in https://github.com/Automattic/vip-support/pull/73/commits/1c0797edf8b3aa4a0ad5569b20a3b39b352a2859 and the regex test in https://github.com/Automattic/vip-support/pull/73/commits/5e8deb8f967deec1f88f55fc3b6fcafd5268cfd6